code/modules/atmospherics/machinery/portable/pipe_scrubber.dm

```diff
@@ -0,0 +1,166 @@
+/obj/machinery/portable_atmospherics/pipe_scrubber
+	name = "pipe scrubber"
+	desc = "A machine for cleaning out pipes of lingering gases. It is a huge tank with a pump attached to it."
+	icon_state = "pipe_scrubber"
+	density = TRUE
+	max_integrity = 250
+	volume = 200
```

Comment:
thoughts on increasing the volume on this? It's effectively the working capacity of the machine, 200L would limit it to the effectiveness of an RPD gas filter. 1000 would be parity with a portable pump, for example.

Reply:
It is supposed to work with the same effectiveness as the floor filter. No need to change.